### DISCUSSION TITLE
Camera-orbit: Consistent 3D Model View Across Devices

### DISCUSSION THREAD

**Discussion by MarBar81** [google/model-viewer#4591] [Q&A]
Hi, I'm trying to figure out how to ensure that the view of the 3D model always maintains the same position, calculated by the camera-orbit, regardless of the device it is displayed on.

The camera-orbit attribute is set as follows:
camera-orbit="calc(-1.5rad + env(window-scroll-y) * 4rad) calc(0deg + env(window-scroll-y) * 180deg) calc(5m - env(window-scroll-y))"

However, when I view the page on a mobile device, the view differs due to variations in window-scroll-y values caused by responsiveness. The model-viewer module's initial position is different from the top of the page, resulting in varying window-scroll-y values and subsequently altering the view.

Could anyone please provide guidance on how to address this issue?

**Comment by elalish** [comment]
Hmm, that's a good question. Maybe get the initial window scrollY position and subtract it in your `calc` statements? I haven't actually tried anything quite like this, so I'll be interested to hear if you get anything working.
